NEWS.md

```diff
@@ -267,7 +267,13 @@ This section lists changes that do not have deprecation warnings.
   * The return type of `reinterpret` has changed to `ReinterpretArray`. `reinterpret` on sparse
     arrays has been discontinued.
 
-  * `Base.find_in_path` is now `Base.find_package` or `Base.find_source_file` ([#24320])
+  * `Base.find_in_path` is now `Base.find_package` or `Base.find_source_file` ([#24320]).
+
+  * `finalizer` now takes functions or pointers as its first argument, and the object being 
```

Comment:
All of these lines have trailing whitespace, which is causing the CI failures. (It looks like we forgot to set up `make check-whitespace` on Circle CI.)